terraform/040-id-broker/vars.tf

```diff
@@ -194,8 +194,8 @@ variable "memory" {
 
 variable "memory_cron" {
   type        = "string"
-  description = "Amount of memory to allocate to cron container, recommend '64' for production"
-  default     = "50"
+  description = "Amount of memory to allocate to cron container, recommend '200' for production"
```

Comment:
```suggestion
  description = "Amount of memory to allocate to cron container, recommend '200' for more than a thousand users"
```

Reply:
Good suggestion to make it about the data size. Incidentally, I think one of the production instances fails with only 500 active users.